presto-hive/src/main/java/com/facebook/presto/hive/NoAccessControl.java

```diff
@@ -59,6 +60,13 @@ public void checkCanRenameTable(Identity identity, SchemaTableName tableName, Sc
     }
 
     @Override
+    public void checkCanRenameColumn(Identity identity, SchemaTableName tableName)
+    {
+        // column rename is not supported in Hive
+        denyRenameColumn(tableName.toString());
```

Comment:
it seems like this isn't a security error to me. It's just that we haven't implemented it yet